src/helpers/getDeckAdvice/advice/OBSIDIAN_BUTCHERS/index.js

```diff
@@ -9,27 +11,38 @@ export default cards => {
   const hasObsidianButchers = cardIds.includes('F17')
   const hasRainOfFrogs = cardIds.includes('F8')
   const hasAzureHatchers = cardIds.includes('F10')
+  const hasLudicMatriarchs = cardIds.includes('N43')
+  let hasLowLvlMatriarchs = false
+  if (hasLudicMatriarchs) {
+    hasLowLvlMatriarchs = cards.find( card =>  card.id === 'N43' ).level <= 2 ? true : false
+  }
```

Comment:
You can simplify this code a little bit by looking for the Ludic Matriarchs card in the deck. If it’s found, the constant will hold the card object, including its level—otherwise `undefined`.
```suggestion
  const LudicMatriarchs = cards.find(card => card.id === 'N43')
```
Then on L23, you can do:

```js
const dragonMsg = (LudicMatriarchs?.level <= 2 && numDragons <= 3) ? '…' : ''
```

`?.` is the optional chaining operator. It means it will not crash if trying to read the `level` property on an `undefined` value. It’s similar to `LudicMatriarchs && LudicMatriarchs.level`.